### ISSUE TITLE
tvOS apps built with Xcode 9 that use Alamofire won't launch on tvOS 10.2

### ISSUE THREAD

**Issue by AtomicCat** [Alamofire/Alamofire#2164]
I filed this bug with Apple today. Probably an Xcode issue, but I thought I'd log it here as well for completeness.

>If you build a tvOS application that links against Alamofire using Xcode 9 9M136h (either swift 3.2 or 4.0) and attempt to run on tvOS 10.2, the app crashes and the following is shown in the console:

> dyld: Symbol not found: _OBJC_CLASS_$_NSURLSessionStreamTask
  Referenced from: /private/var/containers/Bundle/Application/A6598C3C-74B0-4464-BE26-A82F9FF914AC/TvTest.app/Frameworks/Alamofire.framework/Alamofire
  Expected in: /System/Library/Frameworks/Foundation.framework/Foundation
 in /private/var/containers/Bundle/Application/A6598C3C-74B0-4464-BE26-A82F9FF914AC/TvTest.app/Frameworks/Alamofire.framework/Alamofire

>The same project built with Xcode 8.3.3 works as expected.

Basically, create a new tvOS project. Add AF via Cocoapods and import into a file (don't need to call into AF). Attempt to run on a tvOS 10.2 device.

**Comment by jshier** [comment]
We've seen errors like this previous for Xcode 9 beta 1. Clear your derived data and try again. #2142 was similar.

**Comment by AtomicCat** [comment]
I deleted my DerivedData folder each time switching from XC 9 to 8 and back to 9. That did not help.

**Comment by cnoon** [comment]
Re-opening @jshier...we'll need to dig in here a bit to see what's going on.

**Comment by crucibond** [comment]
I have the same issue. I have cleared the dervied data folder and it doesnt seems to help either.

dyld: Symbol not found: _OBJC_CLASS_$_NSURLSessionStreamTask
  Referenced from: /private/var/containers/Bundle/Application/83E6570D-9EB6-4B6C-8871-755942F43B8C/fodmapRevamp.app/Frameworks/Alamofire.framework/Alamofire
  Expected in: /System/Library/Frameworks/Foundation.framework/Foundation
 in /private/var/containers/Bundle/Application/83E6570D-9EB6-4B6C-8871-755942F43B8C/fodmapRevamp.app/Frameworks/Alamofire.framework/Alamofire

**Comment by jshier** [comment]
@AtomicCat @crucibond Is this still and issue for you with Xcode 9 beta 2?

**Comment by jshier** [comment]
After investigating this further, I can replicate it on a variety of project configurations. Manually integrated and CocoaPods integrated projects fail the same way, but only on tvOS. My test projects for iOS or macOS work fine. I suggest we all file bugs with Apple and attach a test project. I'm guessing something might be wrong with the Foundation framework on tvOS.

**Comment by jshier** [comment]
@AtomicCat What is the issue number of the bug you filed? I'd like to duplicate it.

**Comment by AtomicCat** [comment]
Yes, I tried it over the holiday and it was still a problem.

radar://32828698

I attached a small test project showing the issue when I submitted that.

**Comment by AtomicCat** [comment]
Apple just replied that my ticket was a closed as a duplicate of 32298344

**Comment by crucibond** [comment]
@jshier  yes it I still an issue. I am wondering why this is happening.

**Comment by jshier** [comment]
I was successfully able to run my test project using Xcode 9 beta 3, so I believe this issue has been resolved @AtomicCat @crucibond.
